lib/cocoapods/installer/target_installer/aggregate_target_installer.rb

```diff
@@ -102,16 +102,11 @@ def create_bridge_support_file
         end
       end
 
-      # Creates a script that copies the resources to the bundle of the client
-      # target.
+      # Uniqued Resources grouped by config
       #
-      # @note   The bridge support file needs to be created before the prefix
-      #         header, otherwise it will not be added to the resources script.
+      # @return [Hash{ Symbol => Array<Pathname> }]
       #
-      # @return [void]
-      #
-      def create_copy_resources_script
-        path = target.copy_resources_script_path
```

Comment:
This removed line makes every other test to fail.
PR #3337 adds it back.
Sorry again.